.circleci/config.yml

```diff
@@ -15,18 +15,23 @@ defaults: &defaults
 
 jobs:
   test-node6:
+    <<: *defaults
```

Comment:
I've also moved this from below the `docker` line because it doesn't seem as clear that "docker" is overriding the inherited defaults (which contain docker).